src/app/register/register.component.ts

```diff
@@ -9,6 +10,20 @@ import { FormControl, FormGroup, Validators } from '@angular/forms';
 export class RegisterComponent {
   registerForm = new FormGroup({
     userName: new FormControl('', Validators.required),
-    password: new FormControl('', Validators.required)
+    password: new FormControl('', Validators.required),
   });
+
+  constructor(private registerService: LoginApiService) {}
```

Comment:
We should use correct naming for each service to make sure it's not confusing which service is being used

Reply:
Thanks, updated!